lib/folio/library_store.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Folio
+  # A cache of library data
+  class LibraryStore
```

Comment:
Can we just call this `Folio::Library` to match all our other code bases?

Reply:
Sure, done.